src/changelog/comment_changes.rs

```diff
@@ -89,6 +89,10 @@ pub struct CommentChanges {
     )]
   output_directory: String,
 
+  /// The commit to stop at.
+  #[arg(long, short = 'S')]
+  stop_at: Option<git2::Oid>,
```

Comment:
This is the CLI option which will Aeruginous cause to stop.  The long name `stop-at` was chosen for clarity, the short name `S` is intended as visual indication ("stop sign").  We use the commit SHA parser of the Git2 crate, so Aeruginous will alert us when an invalid SHA was given.

Reply:
Sure, the variable name make sense and having Clab and Git2 doing the validation is great!